showcase/treemap/zoomable-treemap.js

```diff
@@ -0,0 +1,184 @@
+// Copyright (c) 2016 - 2017 Uber Technologies, Inc.
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+
+import React from 'react';
+import Treemap from 'treemap';
+import D3FlareData from '../datasets/d3-flare-example.json';
+import ShowcaseButton from '../showcase-components/showcase-button';
+import {foldChilndrens, findBranchByOmen, colorFromValue} from './tree-tools'
+
+const DEFAUL_ROOT_NAME = 'Root'
+const MODE = [
+  'binary',
+  'circlePack',
+  'partition',
+  'partition-pivot',
+  'squarify',
+  'resquarify',
+  'slice',
+  'dice',
+  'slicedice'
+];
+
+export default class ZoomableTreemapExample extends React.Component {
```

Comment:
nit: move the class to be the last thing in the file

Reply:
Ok, done in c489372a29f89e92510655e56d62ccc9e63dba0c